### DISCUSSION TITLE
Add file extension (.md) to wiki link?

### DISCUSSION THREAD

**Discussion by superhu4** [notaapp/talk#251] [Questions] [answered]
Is it possible to _automatically_ add the file extension (.md) when adding a wiki link?

(I am able to _manually_ add it after the wiki link is created.)

**Comment by astoilkov** [comment, marked as answer]
Currently, it isn't possible.

However, we are considering making it possible through our extensions API. We have a lot of use cases for such extensibility and we are pretty sure we will do it in one of our next releases.
